controller/app/validators/key_content_validator.rb

```diff
@@ -0,0 +1,15 @@
+class KeyContentValidator < ActiveModel::Validator
+   def validate(record)
+     key = Net::SSH::KeyFactory.load_data_public_key(record.type + " " + record.content)
```

Comment:
We need some exception handling here. I'm hacking on something, but if the data passed in is not a key that Net::SSH can parse we're getting 500 errors rather than properly handling bad keys via the existing validators.

I'm thinking if the key is of a type we have a minimum size for and it fails to load the key we should return the same code we would as if it didn't meet the length requirements but with a message that the key length couldn't be validated.

@oybed were you using any particular version of net-ssh? If we can use 2.7.0 it'd be best as that's what we already have in the channels.